### PR TITLE
CHANGELOG for upcoming v0.58.0 release

### PR DESCRIPTION
**Pull request by arielshaqed**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,19 @@
 # Changelog
 
+## v.58.0 - 2022-01-26
```
Comment (guy-har) [inline CHANGELOG.md:3]:
```suggestion
## v0.58.0 - 2022-01-26
```

**Review by guy-har — APPROVED** [review]
LGTM just add the `0`

**Comment by arielshaqed** [comment]
Thanks!

This whole "decimal system" thing evidently confuses me.  One day I shall learn...

**Comment by arielshaqed** [comment]
Pulling once tests pass.

**Pull request merged**